_inc/client/performance/search.scss

```diff
@@ -0,0 +1,12 @@
+@import '../scss/variables/colors';
+
+.jp-form-search-setting-progress {
+	margin-bottom: 1em;
+
+	.progress-bar {
+		width: 75%;
```

Comment:
This looks a bit short to represent complete indexing.

Reply:
It does look a bit awkward in its current state. @keoshi, do you have any thoughts on this?

Currently, the progress bar looks like this in wp-admin:
<img width="1057" alt="Screen Shot 2020-04-17 at 3 23 10 PM" src="https://user-images.githubusercontent.com/4044428/79615176-5d42f700-80bf-11ea-8e4e-daf0ae15af0c.png">

The design mock looks like this:
![image](https://user-images.githubusercontent.com/390760/78982999-39117400-7b1b-11ea-997e-c6c8a0067797.png)